docs/design/consistency.md

```diff
@@ -1,21 +1,19 @@
-# Integration
+# Consistency
 
 TigerBeetle is designed to guard against bugs not only in its
 [own code](https://github.com/tigerbeetledb/tigerbeetle/blob/main/docs/TIGER_STYLE.md), but
 at the boundaries, in the application code which interfaces with TigerBeetle.
 This is exhibited by the client's API design, which may be surprising (see [Retries](#retries)) when
 contrasted with a more conventional database.
 
-Strict consistency guarantees (at the database level) simplify application logic and error handling
+Strict consistency guarantees (at the database level) simplify application logic and push error handling
```

Comment:
I think this was correct before. 

By "simplify error handling farther up the stack" I mean that the stronger consistency shrinks the space of possible errors. For example, thanks to "A client session can consider a request executed when it receives a reply for the request.", a client knows that once it receives a reply, the corresponding request is permanently committed — it doesn't need to worry that a crash or something could undo the change.